### PR TITLE
Add Travis CI, Release, devDependencies, and Gitter badges.

### PR DESCRIPTION
**Pull request by WarenGonzaga**
Add this project on Travis CI!
Also update the README with Github Badges!

### CODE REVIEW

**Comment by eltonmesquita** [comment]
This is a very much welcomed commit. +1 for get this merged!

**Comment by WarenGonzaga** [comment]
Thanks you buddy!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+*Just-add-water CSS animation* [![GitHub forks](https://img.shields.io/github/forks/daneden/animate.css.svg?style=social&label=Fork&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub stars](https://img.shields.io/github/stars/daneden/animate.css.svg?style=social&label=Star&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub watchers](https://img.shields.io/github/watchers/daneden/animate.css.svg?style=social&label=Watch&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub followers](https://img.shields.io/github/followers/daneden.svg?style=social&label=Follow&maxAge=2592000)](https://github.com/daneden)
```
Comment (daneden) [inline README.md]: I feel like the GitHub buttons are unnecessary (assuming that most people are hitting this page on GitHub anyway!)

Comment (WarenGonzaga) [inline README.md]: sure buddy I'll do it!

**Comment by WarenGonzaga** [comment]
Hey @daneden  I've removed the github buttons! what do you think?

**Comment by rjp2525** [comment]
Are the dependencies, built with and license badges necessary? There aren't any normal dependencies, gulp is ehh (given there's a gulpfile it should be obvious) and the license is shown at the top right of repositories by default now. There's too many badges, causing a new line:

Badges

I think release, build status, dev dependencies and gitter should be plenty.

**Comment by daneden** [comment]
Agree with @rjp2525. @WarenGonzaga mind reducing the badges to just release, build status, dev dependencies, and gitter?

**Comment by WarenGonzaga** [comment]
Hello guys! Sorry for that I am abusive with github badges haha! I will do that buddies!

**Comment by WarenGonzaga** [comment]
I've removed the extra badges on your ReadMe

**Comment by daneden** [comment]
Awesome. Thanks!

**Comment by WarenGonzaga** [comment]
No problem! @daneden you should checkout the gitter chat for animate.css!

**Pull request merged**